include/deal.II/base/parameter_acceptor.h

```diff
@@ -495,6 +498,7 @@ class ParameterAcceptor : public Subscriptor
    */
   void leave_my_subsection(ParameterHandler &prm);
 
+private:
```

Comment:
Why is this change necessary? I don't see it called by `ParameterAcceptorProxy`.

Reply:
I have a few other tests (on my personal classes, not in deal.II) that need to know the exact subsection of the given class. Since the `get_section_name` is public, I thought this should be public too, to allow more complex situations (e.g., you need to change the value of a parameter after some conditions are met, and you want to make sure you enter in the right section).